### ISSUE TITLE
Some questions about the test set

### ISSUE THREAD

**Issue by wanghao15536870732** [VIPriors/vipriors-challenges-toolkit#17]
Can we use pseudo-labeling to improve our accuracy?

**Comment by rjbruin** [comment]
~~Only with the same model. We do not allow using any kind of pre-training, external model checkpoints or using any data external to the challenge. But if you want to train a model from scratch to make pseudo-labels on the challenge data, you can!~~

Please see the revised policy below!

**Comment by LcenArthas** [comment]
I am curious to know if I can use testing data as pseudo-labels and include in my training. Because I saw in another question that your answer is can not use the unlabeled data.
[https://github.com/VIPriors/vipriors-challenges-toolkit/issues/7](url)
![1631536986(1)](https://user-images.githubusercontent.com/48119351/133085549-7de7c5c4-1eda-4c58-8c0f-abc0a4ac68e3.png)

Thank you.

**Comment by rjbruin** [comment]
Thank you for spotting that. Though that question was answered for last year's challenge, we should have answered consistently with last year's challenge.

So, I'm afraid we have to **revert** the earlier decision to allow using the test set for pseudo-labels:

**It is not allowed to use the test set for anything else than generating predictions to submit to the evaluation server.** This covers pseudo-labeling, using test set statistics, etc.

We are sorry for the confusion caused. If @wanghao15536870732 or anyone else has spent test set evaluation tries on models that now do no longer qualify, please reach out through vipriors-ewi@tudelft.nl.

**Comment by Wprofessor** [comment]
> Thank you for spotting that. Though that question was answered for last year's challenge, we should have answered consistently with last year's challenge.
> 
> So, I'm afraid we have to **revert** the earlier decision to allow using the test set for pseudo-labels:
> 
> **It is not allowed to use the test set for anything else than generating predictions to submit to the evaluation server.** This covers pseudo-labeling, using test set statistics, etc.
> 
> We are sorry for the confusion caused. If @wanghao15536870732 or anyone else has spent test set evaluation tries on models that now do no longer qualify, please reach out through [vipriors-ewi@tudelft.nl](mailto:vipriors-ewi@tudelft.nl).

Hello, I would like to ask how do you determine if pseudo tags are used to train the data?

**Comment by rjbruin** [comment]
That's a good question! We require all competitors that want to qualify for the final leaderboards to submit a technical report, detailing their approach and stating that no external data was used.
